src/main/kotlin/uk/gov/gdx/datashare/services/CognitoService.kt

```diff
@@ -0,0 +1,78 @@
+package uk.gov.gdx.datashare.services
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import software.amazon.awssdk.services.cognitoidentityprovider.CognitoIdentityProviderClient
+import software.amazon.awssdk.services.cognitoidentityprovider.model.CreateUserPoolClientRequest
+import software.amazon.awssdk.services.cognitoidentityprovider.model.ExplicitAuthFlowsType
+import software.amazon.awssdk.services.cognitoidentityprovider.model.OAuthFlowType
+import software.amazon.awssdk.services.cognitoidentityprovider.model.TokenValidityUnitsType
+import uk.gov.gdx.datashare.enums.CognitoClientType
+import uk.gov.gdx.datashare.models.CognitoClientRequest
+import uk.gov.gdx.datashare.models.CognitoClientResponse
+
+@Service
+class CognitoService(
+  @Value("\${cognito.user-pool-id}") val userPoolId: String,
+  @Value("\${cognito.enabled}") val enabled: Boolean,
+  @Value("\${cognito.acquirer-scope}") val acquirerScope: String,
+  @Value("\${cognito.supplier-scope}") val supplierScope: String,
+  @Value("\${cognito.admin-scope}") val adminScope: String,
```

Comment:
Can we make these nullable and default to null so we don't need to set the in application-dev? Seems weird to need to set dummy values

Reply:
Sure